usecases/mail-sender/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+mail.from = noreply@notification.us-ashburn-1.oci.oraclecloud.com
+mail.fromName = OCI Notification
+mail.smtp.user = ocid1.user.oc1..aaaaaaaa3jfondpbrtjds64hk6jfzvtmgxqr7k4wni4t4jergmxe627hf2bq@ocid1.tenancy.oc1..aaaaaaaasu7rvefmsyk5kqczfmdqi5clpddejfjk2attdqnk6sbk72wajq5q.7l.com
+mail.smtp.password = 1#rMe.A(seqHm{vF[Jkt
+mail.smtp.host = smtp.email.us-ashburn-1.oci.oraclecloud.com
```

Comment:
Maybe an environment variable could be used for the OCI region (or commit this as using a placeholder value that requires the user to update - even if it fails as-is, requiring them to set it to the correct value).